### PR TITLE
Switch from Modulefile to metadata.json

### PR DESCRIPTION
**Pull request by roidelapluie**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "dobbymoodge-acl",
+  "version": "0.0.2",
+  "author": "dobbymoodge",
```
Comment (dobbymoodge) [inline metadata.json]: Can multiple authors be specified? I believe you deserve to be in here.

File: metadata.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "dobbymoodge-acl",
+  "version": "0.0.2",
```
Comment (dobbymoodge) [inline metadata.json]: May as well make this "0.0.3" huh?

**Comment by roidelapluie** [comment]
@dobbymoodge What about this?

**Comment by dobbymoodge** [comment]
@roidelapluie Looks good to me!

**Comment by roidelapluie** [comment]
now, git-tag 0.0.3 and publish to the forge. this can be automated with travis

https://configdocs.web.cern.ch/configdocs/upstream/toforge.html

**Pull request merged**